apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -210,7 +215,20 @@ class RealGrunnlagService(
     }
 
     override fun hentGrunnlagAvType(sak: Long, opplysningstype: Opplysningstype): Grunnlagsopplysning<JsonNode>? {
-        return opplysningDao.finnNyesteGrunnlag(sak, opplysningstype)?.opplysning
+        val opplysning = opplysningDao.finnNyesteGrunnlag(sak, opplysningstype)?.opplysning
+        if (opplysning != null || opplysningstype != Opplysningstype.HISTORISK_FORELDREANSVAR) {
```

Comment:
Kanskje vi kunne fått splittet historisk i en egen metode her?

Reply:
Ja, det er nok rimeligere enn å ha "merkelig" oppførsel for denne samlede metoden